src/lib/hooks/use-body-focus-on-route-change.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
+
+/**
+ * Resets focus to the `body` element on route changes.
+ *
+ * Note that the `body` element must have `tabindex={-1}` to be focusable.
+ */
+export default function useBodyFocusOnRouteChange() {
+	const router = useRouter()
+
+	useEffect(() => {
+		const handleRouteChange = () => document.body.focus()
```

Comment:
@zchsh In order to fix our issues. (See review video.)

We should:
1. Create an empty div in _document.tsx with the id `announce-route-change` with the aria-live="polite" set on it.
2. On route change put the text "Page changed to <new page title>." in the `announce-route-change` div
3. Check if focus already exists, if it doesn't focus on the body

I believe this will fix all of our current issues.